src/System.Text.Encoding/tests/EncodingTestHelpers.cs

```diff
@@ -23,7 +42,7 @@ public static void GetByteCount(Encoding encoding, string chars, int index, int
             Assert.Equal(expected, encoding.GetByteCount(charArray, index, count));
         }
         
-        public static void GetBytes(Encoding encoding, string source, int index, int count, byte[] bytes, int byteIndex, byte[] expectedBytes)
+        private static void GetBytes(Encoding encoding, string source, int index, int count, byte[] bytes, int byteIndex, byte[] expectedBytes)
```

Comment:
I'm not really sure I understand you - tests for the output buffers are already done in one place - they're called from `GetBytes(Encoding encoding, string source, int index, int count, byte[] bytes, int byteIndex, byte[] expectedBytes)`, calling the function `VerifyGetBytes(byte[] bytes, int byteIndex, int byteCount, byte[] originalBytes, byte[] expectedBytes)``